BUILD.gn

```diff
@@ -0,0 +1,507 @@
+    "runtime/browser/devtools/xwalk_devtools_manager_delegate.h",
+    "runtime/browser/geolocation/xwalk_access_token_store.cc",
+    "runtime/browser/geolocation/xwalk_access_token_store.h",
+    "runtime/browser/image_util.cc",
+    "runtime/browser/image_util.h",
+    "runtime/browser/media/media_capture_devices_dispatcher.cc",
+    "runtime/browser/media/media_capture_devices_dispatcher.h",
+    "runtime/browser/runtime.cc",
+    "runtime/browser/runtime.h",
+    "runtime/browser/runtime_download_manager_delegate.cc",
+    "runtime/browser/runtime_download_manager_delegate.h",
+    "runtime/browser/runtime_file_select_helper.cc",
+    "runtime/browser/runtime_file_select_helper.h",
+    "runtime/browser/runtime_geolocation_permission_context.cc",
+    "runtime/browser/runtime_geolocation_permission_context.h",
+    "runtime/browser/runtime_javascript_dialog_manager.cc",
+    "runtime/browser/runtime_javascript_dialog_manager.h",
+    "runtime/browser/runtime_network_delegate.cc",
+    "runtime/browser/runtime_network_delegate.h",
+    "runtime/browser/runtime_notification_permission_context.cc",
+    "runtime/browser/runtime_notification_permission_context.h",
+    "runtime/browser/runtime_platform_util.h",
+    "runtime/browser/runtime_platform_util_android.cc",
+    "runtime/browser/runtime_platform_util_aura.cc",
+    "runtime/browser/runtime_platform_util_linux.cc",
+    "runtime/browser/runtime_platform_util_mac.mm",
+    "runtime/browser/runtime_platform_util_win.cc",
+    "runtime/browser/runtime_quota_permission_context.cc",
+    "runtime/browser/runtime_quota_permission_context.h",
+    "runtime/browser/runtime_resource_dispatcher_host_delegate.cc",
+    "runtime/browser/runtime_resource_dispatcher_host_delegate.h",
+    "runtime/browser/runtime_resource_dispatcher_host_delegate_android.cc",
+    "runtime/browser/runtime_resource_dispatcher_host_delegate_android.h",
+    "runtime/browser/runtime_select_file_policy.cc",
+    "runtime/browser/runtime_select_file_policy.h",
+    "runtime/browser/runtime_ui_delegate.cc",
+    "runtime/browser/runtime_ui_delegate.h",
+    "runtime/browser/runtime_ui_delegate_desktop.cc",
+    "runtime/browser/runtime_ui_delegate_desktop.h",
+    "runtime/browser/runtime_url_request_context_getter.cc",
+    "runtime/browser/runtime_url_request_context_getter.h",
+    "runtime/browser/speech/speech_recognition_manager_delegate.cc",
+    "runtime/browser/speech/speech_recognition_manager_delegate.h",
+    "runtime/browser/ssl_error_page.cc",
+    "runtime/browser/ssl_error_page.h",
+    "runtime/browser/storage_component.cc",
+    "runtime/browser/storage_component.h",
+    "runtime/browser/sysapps_component.cc",
+    "runtime/browser/sysapps_component.h",
+    "runtime/browser/ui/color_chooser.cc",
+    "runtime/browser/ui/color_chooser.h",
+    "runtime/browser/ui/color_chooser_android.cc",
+    "runtime/browser/ui/color_chooser_aura.cc",
```

Comment:
Yes, or at least they should (they were being automatically excluded in gyp anyway).